source/content/guides/wordpress-composer/06-bedrock-composer.md

```diff
@@ -0,0 +1,81 @@
+---
```

Comment:
This page should be removed since #7630 has been merged.

Reply:
@jspellman814 In light on your comments, I think we can close out this PR. The current page nests the GitHub Actions, etc under one page. [Current guide](https://pantheon.io/docs/guides/wordpress-composer)